doc/Command-Reference.md

```diff
@@ -220,6 +220,7 @@ This command lists all the possible configuration commands at the top level.
     aaa                    AAA command line
     acl                    ACL-related configuration tasks
     bgp                    BGP-related configuration tasks
+    change-hostname        Change Hostname on a SONiC device without...
```

Comment:
Please also change this to reflect the new subcommand name.

Also please update the PR title to reflect this change.

Reply:
_modified as requested._